frontend/talentsearch/src/js/components/ApplicationPageWrapper/Step.tsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import Link from "@common/components/Link";
+
+export interface StepProps {
+  path: string;
+  label: string;
```

Comment:
label should probably be a React.ReactNode type I think
so that you can pass in intl objects for intl-ing,

Reply:
Yeah, I forgetting sometimes we want to pass HTML to these 😓 

Fixed in [update step label prop type](https://github.com/GCTC-NTGC/gc-digital-talent/pull/3565/commits/4ef045a59d935864dab64b64165d2e7042d9df79)